taichi/program/kernel.cpp

```diff
@@ -145,10 +169,42 @@ void Kernel::set_arg_int(int i, int64 d) {
   }
 }
 
+int64 Kernel::get_arg_int(int i) { // TODO: will consider uint?
```

Comment:
Does unsigned/signed extension matter for get? I just copied your set_arg_int.

Reply:
That should be fine.